src/Book.js

```diff
@@ -0,0 +1,24 @@
+import {useState} from 'react';
+import './Book.css';
+import BookCheckBox from './BookCheckBox';
+
+const Book = props => {
+    const [checked, setChecked] = useState('');
+
+    const bookCheckHandler = checked => {
+        setChecked(checked);
+    }
+    
+    return (
+        <div className={`${props.className} ${checked ? "book_selected" : ""}`}>
```

Comment:
This conditional styling is fine, however I can suggest to look into [classnames](https://www.npmjs.com/package/classnames) package to make it more flexible for further development.